app/models/infrastructure_facades/plgrid/grid_schedulers/qsub.rb

```diff
@@ -168,7 +168,7 @@ def job_pbs_file(sm_uuid)
     # Subinfrastructures are distinguished by:
     #  * grant ids
     def get_subinfrastructures(user_id)
-      PlGridFacade.retrieve_grants(GridCredentials.find_by_user_id(user_id)).flat_map do |grant_id|
+      PlGridFacade.retrieve_grants(GridCredentials.where(user_id: user_id).first).flat_map do |grant_id|
```

Comment:
Dlaczego to zostało tak zmienione, czy pod metodą find_by_user_id nie dzieje się dokładnie to samo? Chyba użyte jest method_missing.